doc/quick_start/contributor-guide.rst

```diff
@@ -42,15 +42,14 @@ As a Linux developer, you should also be familiar with common development tools
 Logging Bugs and Enhancement Requests
 =====================================
 
-Before starting on a patch, first check in `Ostro Project JIRA`_ to see what’s been reported on the issue you’re addressing. 
+Before starting on a patch, first check in `Ostro Project Issues`_ to see what’s been reported on the issue you’re addressing. 
 Have a conversation on the mailing list to see what others think of your issue (and proposed solution). 
 You may find others that have encountered the issue you're finding, or with similar ideas for changes or additions. 
 If you don't, send a message to the mailing list to introduce and discuss your idea with the development community.
 
-After logging into JIRA, click “Issues” to search for open items. To avoid duplication, it's always a good practice 
+After access the ``ostro-os`` GitHub repo, click “Issues” to search for open (and closed) items. To avoid duplication, it's always a good practice 
```

Comment:
Shouldn't it say "After accessing..." (instead of access) to be grammatically correct?